templates/corosync.j2

```diff
@@ -0,0 +1,49 @@
+totem {
+        version: {{ ha_cluster_corosync_params.totem.version | default(2) }}
+        token: {{ ha_cluster_corosync_params.totem.token | default(30000) }}
+        consensus: {{ ha_cluster_corosync_params.totem.consensus | default(36000) }}
+        token_retransmits_before_loss_const: {{ ha_cluster_corosync_params.totem.token_retransmits_before_loss_const | default(6) }}
+        secauth: {{ ha_cluster_corosync_params.totem.secauth | default("on") }}
+        crypto_cipher: {{ ha_cluster_corosync_params.totem.crypto_cipher | default("aes256") }}
+        crypto_hash: {{ ha_cluster_corosync_params.totem.crypto_hash | default("sha1") }}
+        clear_node_high_bit: {{ ha_cluster_corosync_params.totem.clear_node_high_bit | default("yes") }}
+        rrp_mode: {{ ha_cluster_corosync_params.totem.rrp_mode | default("passive") }}
+
+        interface {
+                ringnumber: 0
+                bindnetaddr: {{ ansible_facts['eth0']['ipv4']['address'] }}
+                mcastport: 5405
+                ttl: 1
+        }
+        transport: udpu
```

Comment:
If Suse doesn't support knet, it should be mentioned in the documentation, I guess. Is Suse still running corosync 2.x? It looks like that branch is no longer supported: https://github.com/corosync/corosync/wiki/Releases

Also, it would be nice if `tasks/shell_crmsh/check-and-prepare-role-variables.yml` failed when corosync options not working on Suse are specified.

Reply:
Current version is 2.4.6 and 3.1.x is planned for future.